src/main/webapp/js/instructorStudentEdit.js

```diff
@@ -0,0 +1,16 @@
+$(document).ready(function(){
+    readyStudentEditPage();
+});
+
+function readyStudentEditPage(){
+    
+    $("#button_submit").click(function(){
```

Comment:
If you suggesting an enhancement, you can post it as a new issue for further discussion.